src/wp-includes/widgets/class-wp-widget-custom-html.php

```diff
@@ -69,12 +69,12 @@ public function _register_one( $number = -1 ) {
 		}
 		$this->registered = true;
 
-		wp_add_inline_script( 'custom-html-widgets', sprintf( 'wp.customHtmlWidgets.idBases.push( %s );', wp_json_encode( $this->id_base ) ) );
-
 		// Note that the widgets component in the customizer will also do
 		// the 'admin_print_scripts-widgets.php' action in WP_Customize_Widgets::print_scripts().
 		add_action( 'admin_print_scripts-widgets.php', array( $this, 'enqueue_admin_scripts' ) );
 
+		add_action( 'admin_print_scripts-widgets.php', array( $this, 'enqueue_admin_scripts_after' ) );
```

Comment:
Why a new method? It's on the same action hook, so I think we could just add this one line to the existing `enqueue_admin_scripts()` method.

Reply:
@westonruter Can you provide some context, why was this line `wp_add_inline_script` put here and not in `enqueue_admin_scripts`?

Reply:
Unfortunately the PR in which I originally did the work is no longer around, so I can't see the deeper commit history. However, I can't think of a good reason why this logic is in `_register_one` and isn't just put in the `enqueue_admin_scripts` method along with the other `wp_add_inline_script()` calls.

Reply:
@westonruter I am not seeing a different the markup it generates before and after

Before
```


<script id='custom-html-widgets-js-after'>
--
  | wp.customHtmlWidgets.idBases.push( "custom_html" );
  | wp.customHtmlWidgets.init( {"codemirror":{"indentUnit":2,"indentWithTabs":true,"inputStyle":"contenteditable","lineNumbers":true,"lineWrapping":true,"styleActiveLine":true,"continueComments":true,"extraKeys":{"Ctrl-Space":"autocomplete","Ctrl-\/":"toggleComment","Cmd-\/":"toggleComment","Alt-F":"findPersistent","Ctrl-F":"findPersistent","Cmd-F":"findPersistent"},"direction":"ltr","gutters":["CodeMirror-lint-markers"],"mode":"htmlmixed","lint":true,"autoCloseBrackets":true,"autoCloseTags":true,"matchTags":{"bothTags":true},"tabSize":2},"csslint":{"errors":true,"box-model":true,"display-property-grouping":true,"duplicate-properties":true,"known-properties":true,"outline-none":true},"jshint":{"boss":true,"curly":true,"eqeqeq":true,"eqnull":true,"es3":true,"expr":true,"immed":true,"noarg":true,"nonbsp":true,"onevar":true,"quotmark":"single","trailing":true,"undef":true,"unused":true,"browser":true,"globals":{"_":false,"Backbone":false,"jQuery":false,"JSON":false,"wp":false}},"htmlhint":{"tagname-lowercase":true,"attr-lowercase":true,"attr-value-double-quotes":false,"doctype-first":false,"tag-pair":true,"spec-char-escape":true,"id-unique":true,"src-not-empty":true,"attr-no-duplication":true,"alt-require":true,"space-tab-mixed-disabled":"tab","attr-unsafe-chars":true}} );
  | jQuery.extend( wp.customHtmlWidgets.l10n, {"errorNotice":{"singular":"There is %d error which must be fixed before you can save.","plural":"There are %d errors which must be fixed before you can save."}} );
  | </script>



```

After
```


<script id='custom-html-widgets-js-after'>
--
  | wp.customHtmlWidgets.idBases.push( "custom_html" );
  | wp.customHtmlWidgets.init( {"codemirror":{"indentUnit":2,"indentWithTabs":true,"inputStyle":"contenteditable","lineNumbers":true,"lineWrapping":true,"styleActiveLine":true,"continueComments":true,"extraKeys":{"Ctrl-Space":"autocomplete","Ctrl-\/":"toggleComment","Cmd-\/":"toggleComment","Alt-F":"findPersistent","Ctrl-F":"findPersistent","Cmd-F":"findPersistent"},"direction":"ltr","gutters":["CodeMirror-lint-markers"],"mode":"htmlmixed","lint":true,"autoCloseBrackets":true,"autoCloseTags":true,"matchTags":{"bothTags":true},"tabSize":2},"csslint":{"errors":true,"box-model":true,"display-property-grouping":true,"duplicate-properties":true,"known-properties":true,"outline-none":true},"jshint":{"boss":true,"curly":true,"eqeqeq":true,"eqnull":true,"es3":true,"expr":true,"immed":true,"noarg":true,"nonbsp":true,"onevar":true,"quotmark":"single","trailing":true,"undef":true,"unused":true,"browser":true,"globals":{"_":false,"Backbone":false,"jQuery":false,"JSON":false,"wp":false}},"htmlhint":{"tagname-lowercase":true,"attr-lowercase":true,"attr-value-double-quotes":false,"doctype-first":false,"tag-pair":true,"spec-char-escape":true,"id-unique":true,"src-not-empty":true,"attr-no-duplication":true,"alt-require":true,"space-tab-mixed-disabled":"tab","attr-unsafe-chars":true}} );
  | jQuery.extend( wp.customHtmlWidgets.l10n, {"errorNotice":{"singular":"There is %d error which must be fixed before you can save.","plural":"There are %d errors which must be fixed before you can save."}} );
  | </script>

```

I am not seeing the problem. The inline script doesn't make sense where it was, it only outputs when script it output anyway. 

I think this change is good to go.